docs/backends_sankey.py

```diff
@@ -2,8 +2,28 @@
 
 import plotly.graph_objects as go
 
+
+def to_greyish(hex_code, grey_value=128):
+    hex_code = hex_code.lstrip("#")
+    r, g, b = int(hex_code[0:2], 16), int(hex_code[2:4], 16), int(hex_code[4:6], 16)
+
+    new_r = (r + grey_value) // 2
+    new_g = (g + grey_value) // 2
+    new_b = (b + grey_value) // 2
+
+    new_hex_code = f"#{new_r:02x}{new_g:02x}{new_b:02x}"
+
+    return new_hex_code
+
+
+category_colors = {
+    "Ibis API": "#7C65A0",
+    "SQL code": "#6A9BC9",
+    "DataFrame code": "#D58273",
```

Comment:
Can't comment directly on the image, but the new image colors look a lot better. +1.

Also slight preference for dropping the "code" suffix and just going with "SQL" and "DataFrame" or maybe "SQL" and "DataFrame API" idk.